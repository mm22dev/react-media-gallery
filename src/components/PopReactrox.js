import React from 'react'
import PropTypes from 'prop-types'
import { PopupSettingsProvider } from '../providers/PopupSettings'
import Overlay from './ui/Overlay'
import '../stylesheets/css/components/popreactrox.scss'

const PopReactrox = ({ PRTSettings }) => (
  <PopupSettingsProvider PRTSettings={PRTSettings}>
    <Overlay />
  </PopupSettingsProvider>
)

PopReactrox.propTypes = {
  PRTSettings: PropTypes.object
}

PopReactrox.defaultProps = {
  PRTSettings: {}
}

export default PopReactrox
