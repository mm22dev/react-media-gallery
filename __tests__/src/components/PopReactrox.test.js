import PopReactrox from '../../../src/components/PopReactrox'
import { compose } from 'redux'
import toJSON from 'enzyme-to-json'

const { shallow } = Enzyme

describe('<PopReactrox /> Component ', () => {
  describe('Rendering the UI', () => {
    it('Renders correctly', () => {
      compose(expect, toJSON, shallow)(<PopReactrox />).toMatchSnapshot()
    })
  })
})
