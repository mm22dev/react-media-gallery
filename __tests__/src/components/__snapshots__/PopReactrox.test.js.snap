// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PopReactrox /> Component  Rendering the UI Renders correctly 1`] = `
<PopupSettingsProvider
  PRTSettings={Object {}}
>
  <Overlay />
</PopupSettingsProvider>
`;
